ios/sound.mm

```diff
@@ -0,0 +1,232 @@
+/******************************************************************************\
+ * Copyright (c) 2004-2020
+ *
+ * Author(s):
+ *  Volker Fischer
```

Comment:
I have not written this file. BTW: What is the ending ".mm"?

Reply:
.mm means that it's a objective c++ file --> you can mix c++ and objective c in this file. This is needed (as far as I understand) to access the iOS sound API